### PR TITLE
feat: add manual controller

### PR DESCRIPTION
**Pull request by taikitanaka3**
Signed-off-by: tanaka3 <ttatcoder@outlook.jp>

## Description

add manual controller for rviz plugin

## Related links

this plugin can see how manual driving effects autoware planning control modules
eg)
- plannig module won't die while manual driving 
- trajectory bend shaply

## Tests performed

by psim

https://user-images.githubusercontent.com/65527974/170936118-7af9fe8e-7094-47c1-8f2c-137964e21801.mp4

image

## Notes for reviewers

- use this module for debugging

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by shmpwk — COMMENTED** [review]
Interesting! There are two points I noticed.
1. Correspondent movement of steering handle icon and "steering" botton. 
Currently, they are different movent and the users might feel confusing.
2. The definition of "Set Cruise Velovity"
If I set the velocity "3" km/h, the vehicle exceeds the velocity like 6km/h. I think the velocity is the target velocity and it can exceed the target velocity. But it overly overshoot?

**Comment by taikitanaka3** [comment]
[ebb2092](https://github.com/autowarefoundation/autoware.universe/pull/989/commits/ebb20929a5b2ade543204c1b67de13bf41ac3816)
 is going to fix Issues you mentioned thank you

**Review by shmpwk — APPROVED** [review]

File: common/tier4_control_rviz_plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# tier4_control_rviz_plugin
+
+This package is to add manual controller from rviz plugin
```
[the remark this answers is not in the document]
Comment (shmpwk) [reply common/tier4_control_rviz_plugin/README.md]: I added them.

**Review by shmpwk — APPROVED** [review]
LGTM!

File: common/tier4_control_rviz_plugin/src/tools/manual_controller.cpp [hunk 2]
```diff
@@ -0,0 +1,277 @@
+//
+//  Copyright 2020 Tier IV, Inc. All rights reserved.
```
Comment (taikitanaka3) [inline common/tier4_control_rviz_plugin/src/tools/manual_controller.cpp:2]:
```suggestion
//  Copyright 2022 Tier IV, Inc. All rights reserved.
```
TODO fix licenses

**Comment by yukkysaito** [comment]
@taikitanaka3 
Now, @isamu-takagi is developping new ad-api, so pls add @isamu-takagi as a reviewer.

BTW, why did you need this plugin?

**Comment by taikitanaka3** [comment]
@yukkysaito 
OK I will add 

> BTW, why did you need this plugin?
because in simulation environment vehicle does ideal behavior. so it's difficult to find bug in edge case while operation

File: common/tier4_control_rviz_plugin/src/tools/manual_controller.hpp [hunk 3]
```diff
@@ -0,0 +1,103 @@
+//
+//  Copyright 2020 Tier IV, Inc. All rights reserved.
```
Comment (taikitanaka3) [inline common/tier4_control_rviz_plugin/src/tools/manual_controller.hpp:2]:
```suggestion
//  Copyright 2022 Tier IV, Inc. All rights reserved.
```

**Pull request merged**